File: Array/JavaScript/LargestElement.js
function large(arr, N) {
    max = arr[0]

    for (let i = 0; i < N; i++){
        if (arr[i] > max)
            max = arr[i]
    }
    return max
}

arr = [152,235,26,235,23,25,25,62,5]
N = arr.length
res = large(arr, N)
console.log(res)

File: Array/Python/Basic/LargestElement.py
def large(arr):
    max = arr[0]

    for i in range(0,len(arr)):
        if arr[i] > max:
            max = arr[i]
    return max        
    


arr = [10,134,245,64,37256,477]
res = large(arr)
print(res)